src/ChangeStream.php

```diff
@@ -143,8 +143,8 @@ private function extractResumeToken($document)
         if ($document === null) {
             throw new ResumeTokenException("Cannot extract a resumeToken from an empty document");
         }
-        if ($document instanceof Serializable) {
-            return $this->extractResumeToken($document->bsonSerialize());
+        if ($document instanceof \Serializable) {
+            $this->extractResumeToken($document->bsonSerialize());
```

Comment:
@carusogabriel: This looks incorrect. Is PHPStan complaining that a `void` function is returning the result of another `void` function? If so, I think the correct fix is to add a solitary `return;` on the line below. The point of this is to recurse on the result of `$document->bsonSerialize()`, while this change allows execution of the current function to continue.

Reply:
Likely related to https://github.com/phpstan/phpstan/commit/679ea28f29699f409662cd81bd0414769b9e047e.